lgc/patch/gfx9/chip/llpcGfx9ConfigBuilder.cpp

```diff
@@ -115,7 +115,7 @@ void ConfigBuilder::buildPipelineVsFsRegConfig() // [out] Size of register confi
   if (stageMask & shaderStageToMask(ShaderStageVertex)) {
     buildVsRegConfig<PipelineVsFsRegConfig>(ShaderStageVertex, &config);
 
-    SET_REG_FIELD(&config, VGT_SHADER_STAGES_EN, VS_EN, VS_STAGE_REAL);
+    SET_REG_FIELD(&config, VGT_SHADER_STAGES_EN, VS_EN, Pal::Gfx9::Chip::VS_STAGE_REAL);
```

Comment:
Please add `using namespace Pal::Gfx9::Chip` at the beginning of this file. Thus, we don't have to do so many modifications.